Baikal/PostEffects/post_effect.h

```diff
@@ -49,6 +49,43 @@ namespace Baikal
     class PostEffect
     {
     public:
+
+        enum class ParamType
+        {
+            kFloatVal = 0,
+            kUintVal,
+            kFloat4Val,
+            kStringVal
+        };
+
+        struct ParamValue
```

Comment:
This separate struct is not required, because it's only used as a private member of `Param`.

Reply:
Done